src/tags/channelcategory.js

```diff
@@ -20,9 +20,11 @@ module.exports =
         )
         .whenArgs(0, async (_, context) => (context.channel.parentID || ''))
         .whenArgs('1-2', async (subtag, context, args) => {
-            let channel = Builder.util.parseChannel(context, args[0]);
             let quiet = bu.isBoolean(context.scope.quiet) ? context.scope.quiet : !!args[1];
-            if (typeof channel === 'function') return quiet ? '' : channel(subtag, context);
+            let channel = await Builder.util.parseChannel(context, args[0], quiet);
```

Comment:
In this you pass a Boolean as the `args` parameter of `.parseChannel()`. An object with the quiet property should be passed instead to avoid errors to do with trying to set properties of a Boolean value in `context.getChannel()`. Could be changed to `{quiet}`.